apps/web/components/booking/BookingListItem.tsx

```diff
@@ -666,7 +667,11 @@ const Attendee = (attendeeProps: AttendeeProps & NoShowProps) => {
     attendee: { email: string; noShow: boolean };
     bookingUid: string;
   }) {
-    noShowMutation.mutate({ bookingUid, attendees: [attendee] });
+    const seedData = { bookingUid: bookingUid, attendees: [attendee] };
+    const token = encodeURIComponent(
+      symmetricEncrypt(JSON.stringify(seedData), process.env.CALENDSO_ENCRYPTION_KEY || "")
+    );
+    noShowMutation.mutate({ token });
```

Comment:
Encryption shouldn't happen on client side. Also `process.env.CALENDSO_ENCRYPTION_KEY` is not available in browsers for the same reason.